assets/chezmoi.io/docs/developer-guide/website.md

```diff
@@ -2,9 +2,11 @@
 
 The [website](https://chezmoi.io) is generated with [Material for
 MkDocs](https://squidfunk.github.io/mkdocs-material/) from the contents of the
-`assets/chezmoi.io/docs/` directory. It hosted by [GitHub pages](https://pages.github.com/) from
+`assets/chezmoi.io/docs/` directory. It is hosted by [GitHub pages](https://pages.github.com/) from
 the [`gh-pages` branch](https://github.com/twpayne/chezmoi/tree/gh-pages).
 
+You'll need python3 and Go installed to be able to generate the pages.
```

Comment:
This should be more specific and reiterate the main [developer guide](https://www.chezmoi.io/developer-guide/) page, where Go 1.22 or later is required. `python3` should be styled as `Python 3` and probably be `Python 3.10 or later`.

My recommended reword:

```markdown
To build the website locally, both Go 1.22 (or later) and Python 3.10 (or later) must be installed.
```